core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFeedLoader.java

```diff
@@ -116,10 +119,11 @@ public GtfsFeedContainer loadAndValidate(GtfsInput gtfsInput, GtfsFeedName feedN
         }
         try {
             try {
-                List<Future<GtfsTableContainer>> results = exec.invokeAll(loaderCallables);
-                results.forEach(f -> {
+                exec.invokeAll(loaderCallables).forEach(f -> {
```

Comment:
I think the code would be more clear if `f` was renamed, maybe as `tableAndNoticeContainer` or `container`.

Reply:
This "f" is for future. We start from 

```
TableAndNoticeContainers containers = f.get();
```

and do not use it after that.

Reply:
Oh okay! I missed that one, thanks.